pages/configuration/advanced.md

```diff
@@ -13,9 +13,22 @@ boot medium, `config.txt` - you change boot options simply by editing this file.
 
 __Note:__ You will only find the `config.txt` file after first boot, it can then easily be found in the `resin-boot` partition of the SD card.
 
-#### Post-Boot
+#### Modifying `config.txt` locally after the first boot
 
-The `config.txt` is located in the root of the `resin-boot` partition.
+The `config.txt` is located in the root of the `resin-boot` partition, and you can modify it by mounting the SD card on a computer.
+
+#### Modifying `config.txt` **remotely**
+
+If your device has a Supervisor version above 1.0.0, it has support for modifying the values in `config.txt` remotely using [environment variables](/pages/management/env-vars).
+The variables that start with the `RESIN_HOST_CONFIG_` prefix will be added to the `config.txt` file, also replacing the preexisting values of such variables in the file.
+For example, setting the value of `RESIN_HOST_CONFIG_gpu_mem` to 16 will produce the following entry in `config.txt`:
+
+```
+gpu_mem=16
+```
+
+These variables can be set using the SDK or in the "Advanced section" of the environment variables page.
```

Comment:
The "Advanced" section has been renamed to "Application config variables" in the app page and "Device config variables" in the device page.

Also, you do give a link to the env vars docs, but it might be worthy to mention here that users can provide "defaults" at app-level and override specific variables per device. Maybe something like:

_"Config variables specified for the application apply to all devices belonging to that application, but you may redefine config variables per individual device, that override the ones specified for the application."_

Or if that is too terse, maybe something like:

_"Like environment variables, you can specify both application-wide and per-device config variables, with per-device config variables taking precedence over application-wide config variables of the same name."_

Or whatever :P